eslint-import-resolver.js

```diff
@@ -0,0 +1,20 @@
+const resolve = require('resolve');
+const path = require('path');
+
+module.exports = {
```

Comment:
Why isn't this standard behaviour for `eslint-plugin-import`?

Reply:
@kevinbarabash This blocks importing core modules, e.g., `fs` and `path`, and using `.json` modules, which are supported by Node and standard behavior for `eslint-plugin-import`.